opencti-platform/opencti-front/src/private/components/data/ToolBar.js

```diff
@@ -303,6 +309,44 @@ class ToolBar extends Component {
     this.setState({ displayPromote: false });
   }
 
+  handleCopy() {
```

Comment:
I must join Romuald on his comments, I think this function needs to be redone:

1. You should try not using Ramda
2. You are redoing a lot of computation that is already done in ContainerStixCyberObservable and do not interest ToolBar at all
3. Search doesn't seem to be working

Solution:

- Toolbar should not create a HandleCopy function
- ContainerStixCyberObservable should send the correct function to Toolbar that will be using it
- ContainerStixCyberObservable will use the correct filters (that it is already aware of)
- Cherry on the cake: ContainerStixCyberObservable is a pure function based on the work on Cities and this "handleCopy" method should probably be a Hook which signature we need to define.